applications/tari_app_utilities/src/identity_management.rs

```diff
@@ -203,7 +224,11 @@ pub fn save_as_json<P: AsRef<Path>, T: MessageFormat>(path: P, object: &T) -> Re
             fs::create_dir_all(p).map_err(|e| format!("Could not save json to data folder. {}", e.to_string()))?;
         }
     }
-    fs::write(path.as_ref(), json.as_bytes()).map_err(|e| {
+    let json_with_comment = format!(
+        "// The public address is overwritten by the config/environment variable\n{}",
```

Comment:
I think this comment should probably specify what the ENV var is?

Reply:
I've added TARI_BASE_NODE__<network>__PUBLIC_ADDRESS into the comment.